analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -137,10 +139,6 @@ public void Main(bool boolParameter{additionalParameters})
     public NotImplementedException CreateException() => new NotImplementedException();
     public void InstanceMethod(object parameter = null) {{ }}
     public static void StaticMethod() {{ }}
-
-    private void Tag(string name) {{ }}
-    private static void Tag(string name, object arg = null) {{ }}
-    private static T Unknown<T>() => default;
```

Comment:
I moved the methods further away from the Sample class. I hope the invocations to Tag no longer interfere with our logic now and in the future.